### PR TITLE
move out rollout e2e

### PR DESCRIPTION
**Pull request by ryanzhang-oss**
move out rollout e2e

### CODE REVIEW

File: .github/workflows/go.yml [hunk 1]
```diff
@@ -123,6 +123,62 @@ jobs:
       - name: Clean up testdata
         run: make compatibility-testdata-cleanup
 
+  e2e-rollout-tests:
+    runs-on: aliyun
```
Comment (wonderflow) [inline .github/workflows/go.yml:127]: separate out doesn't help, the CI runs in the same queue  on `aliyun`

Comment (ryanzhang-oss) [reply .github/workflows/go.yml:127]: won't they run in parallel? Also, it's at least good to see that the original e2e failed in this PR.

Comment (wonderflow) [reply .github/workflows/go.yml:127]: They are not parallel in `aliyun` runner

Comment (ryanzhang-oss) [reply .github/workflows/go.yml:127]: now there are more than one machine

**Review by hongchaodeng — APPROVED** [review]
lgtm

**Pull request merged**